Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM php:7.3-fpm
+
+# Arguments defined in docker-compose.yml
+ARG user
+ARG uid
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    libpng-dev \
+    libonig-dev \
+    libxml2-dev \
+    zip \
+    unzip
+
+# Clear cache
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Install PHP extensions
+RUN docker-php-ext-install pdo_mysql mbstring exif pcntl bcmath gd
+
+# Get latest Composer
+COPY --from=composer:2.1 /usr/bin/composer /usr/bin/composer
```

Comment:
Lates composer right?? Make sure it will be always compatible… since composer version is one of the main culprit behind failed deploys

Reply:
i think this one is already correct, it says 2.1 version for the composer and for as far as the build run successfully then its good

Reply:
sorry i'm just opened github again, i forget to change the comment to Composer 2.1, but the version of composer i've declared is 2.1 and when i test run this docker, that's working properly, thankyouu